### PR TITLE
[WIP, ignore] Cleanup in quantized

### PR DESCRIPTION
**Pull request by z-a-f**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#23115 Cleanup in quantized**

### CODE REVIEW

**Comment by zafartahirov** [comment]
This is a draft: local compilation is still running

File: test/common_quantized.py [hunk 1]
```diff
@@ -3,6 +3,26 @@
 """
 import numpy as np
```
Comment (jerryzh168) [inline test/common_quantized.py]: we have numpy here as well

File: test/common_quantized.py [hunk 2]
```diff
@@ -3,6 +3,26 @@
...
+    a = copy.deepcopy(a)
+    if isinstance(a, Iterable):
+        for idx in len(a):
+            a[idx] = _round(a[idx], ndigits)
+        return a
+    else:
+        return round(a, ndigits)
+
 # Quantization references
 def _quantize(x, scale, zero_point, qmin=None, qmax=None, dtype=np.uint8):
```
Comment (jerryzh168) [inline test/common_quantized.py]: I think this is probably not used, you can check

File: test/test_quantized_tensor.py [hunk 3]
```diff
@@ -60,7 +62,7 @@ def test_qtensor_quant_dequant(self):
         zero_point = 2
         qr = torch.quantize_linear(r, scale, zero_point, torch.quint8)
         rqr = qr.dequantize()
-        self.assertTrue(np.allclose(r.numpy(), rqr.numpy(), atol=2 / scale))
+        self.assertAlmostEqual(r.numpy(), rqr.numpy(), delta=2 / scale)
```
Comment (jerryzh168) [inline test/test_quantized_tensor.py]: we probably don't need to convert to numpy in this case, same for the other places.

File: torch/nn/quantized/modules/conv.py [hunk 4]
```diff
@@ -15,8 +15,7 @@
 """Computes the output shape given convolution parameters."""
 def _conv_output_shape(input_size, kernel_size, padding, stride, dilation,
```
Comment (jerryzh168) [inline torch/nn/quantized/modules/conv.py]: is this function used in tests only? if so maybe move it to common_quantized.py?

**Comment by zafartahirov** [comment]
This is low priority -- will get back to this later

**Pull request closed without merging**